setup.py

```diff
@@ -141,20 +150,9 @@ def get_ifconfig():
 setup(name=NAME, author=AUTHOR, author_email=AUTHOR_EMAIL, description=DESCRIPTION,
       license=LICENSE, keywords=KEYWORDS, packages=PACKAGES,
       include_package_data=INCLUDE_PACKAGE_DATA, version=VERSION, entry_points=ENTRY_POINTS,
-      install_requires=INSTALL_REQUIRES)
+      install_requires=INSTALL_REQUIRES, classifiers=CLASSIFIERS, url=URL)
 
 # Get hostapd, dnsmasq or ifconfig if needed
 get_hostapd()
 get_dnsmasq()
 get_ifconfig()
-
-print()
-print("                     _  __ _       _     _     _               ")
-print("                    (_)/ _(_)     | |   (_)   | |              ")
-print("  ((.))    __      ___| |_ _ _ __ | |__  _ ___| |__   ___ _ __ ")
-print(r"    |      \ \ /\ / / |  _| | '_ \| '_ \| / __| '_ \ / _ \ '__|")
-print(r"   /_\      \ V  V /| | | | | |_) | | | | \__ \ | | |  __/ |   ")
-print(r"  /___\      \_/\_/ |_|_| |_| .__/|_| |_|_|___/_| |_|\___|_|   ")
-print(r" /     \                    | |                                ")
-print("                            |_|                                ")
-print("                                                               ")
```

Comment:
Although I'm not the biggest fan when it comes to ASCII art, as I think it makes the software look more like a script,  I think it would be better if it is run every time the program is run if we wanted to include it.